src/main/java/com/example/ajaxonboardingproject/model/MovieSession.kt

```diff
@@ -1,29 +1,18 @@
 package com.example.ajaxonboardingproject.model
 
-import jakarta.persistence.Column
-import jakarta.persistence.Entity
-import jakarta.persistence.FetchType
-import jakarta.persistence.GeneratedValue
-import jakarta.persistence.GenerationType
-import jakarta.persistence.Id
-import jakarta.persistence.JoinColumn
-import jakarta.persistence.ManyToOne
-import jakarta.persistence.Table
+import org.bson.codecs.pojo.annotations.BsonProperty
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
 import java.time.LocalDateTime
 
-@Entity
-@Table(name = "movie_sessions")
+@Document("movie_sessions")
 class MovieSession(
-    @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "movie_id")
     var movie: Movie,
-    @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "cinema_hall_id")
+    @BsonProperty(value = "cinema_hall")

```

Comment:
Why @BsonProperty?

Usually `org.springframework.data.mongodb.core.mapping.Field` annotation is used.

Nevermind. Just rename your db field to `showTime` and remove the annotation.

In mongo we use camel case for names

Reply:
Done!